plugin/config.json

```diff
@@ -53,7 +53,11 @@
       "description": "Log level - info, warn, error or debug",
       "value": "info",
       "Settable": [ "value"]
-	}
-	]
+	},
+	{"name": "VDVS_DISCOVER_VOLUMES",
+      "description": "non-empty if pre-existing volumes need to be discovered ",
```

Comment:
Yes, and on the top of this github does not set proper tabstops. 

Unfortunately I don't know how to set tab->spaces replacement for json editor in Code. 
For everything else I did find it, but for json we'd need suffer :-) unless someone can  point me out how to set it. 
Add ?ts=4  to the diff URL to see it well